grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -704,6 +704,13 @@ class ShipmentService {
             log.info("Checking shipment item ${shipmentItem?.inventoryItem} quantity [" +
                     shipmentItem.quantity + "] <= quantity on hand [" + quantityOnHand + "]")
             if (duplicatedShipmentItemsQuantity > quantityOnHand && origin.supports(ActivityCode.MANAGE_INVENTORY)) {
+                String errorMessage = "Shipping quantity (${shipmentItem.quantity}) can not exceed on hand quantity (${quantityOnHand}) for " +
```

Comment:
I don't think we need this any longer. You can probably pass null or something generic like "Shipping quantity cannot exceed on hand quantity." as the default message.

Reply:
This one is passed as error message